src/codec_rav1e.c

```diff
@@ -49,13 +49,23 @@ static avifBool rav1eSupports400(void)
     return minorVersion >= 4;
 }
 
+static int rav1eQualityToQuantizer(int quality)
+{
+    int quantizer;
+
+    quality = AVIF_CLAMP(quality, 0, 100);
+    quantizer = ((100 - quality) * 255 + 50) / 100;
```

Comment:
The new conversion is not exactly the same as the original conversion, so this will change the encoded output for some quality values. I think this is okay.